themes/smi/assets/scss/styles.scss

```diff
@@ -814,6 +814,19 @@ article.post {
         border-bottom: 3px solid lighten($bluel, 12.5%);
       }
     }
+
```

Comment:
@bridgetkromhout I added two new css selectors which only affect this new blog post. I was at a loss as to how to wrap text around images, align images, and reduce their size with the current classes. I can remove these two selectors, however, there will be a few unwanted oddities. I will happily change to other befitting, existing classes (or none).

 Please advise.

Reply:
I will defer to @flynnduism who will be able to assess whether these items will create any unintended side effects and/or whether they are okay to add.

Reply:
✅ These additions are well scoped to only apply here, having checked it over, these Sass edits are a /lgtm. I have a minor comment regarding the `margin-right` spacing of the image:

<img width="1389" alt="Screen Shot 2021-04-26 at 3 04 23 PM" src="https://user-images.githubusercontent.com/686194/116157218-6e92ec00-a6a1-11eb-97cc-b644e7d5b536.png">

It's very close to the text and could impact on legibility IMO. Not a showstopper, but I'd recommend increasing the `margin-right` value 2x or 3x.

Reply:
Oh, very good. This margin has been increased by 3x now.